stable/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -109,7 +109,7 @@ spec:
       volumes:
         - name: ssl-certs
           hostPath:
-            path: {{ .Values.sslCertPath }}
+            path: {{ .Values.sslCertHostPath }}
```

Comment:
If `sslCertHostPath` isn't defined in `values.yaml` - which is everyone that uses the chart today - this will break once they update to this version. Please correct me of that's wrong.

Therefore I suggest we fallback to `sslCertPath` if `sslCertHostPath` isn't defined, e.g. something like this `{{ .Values.sslCertHostPath || .Values.sslCertPath }}` (I don't know the right syntax, you could check in another chart).

If we leave it like that we should at least bump the chart's minor version.

Reply:
Made the change requested for fallback